educator/src/Dscp/Educator/Web/Educator/API.hs

```diff
@@ -40,7 +40,7 @@ protectedEducatorAPI = Proxy
 
 data EducatorApiEndpoints route = EducatorApiEndpoints
     {
-      -- * Students
```

Comment:
Those commits are invalid. There are much more of these, only cleaned up several files before switching to `doHaddock = false;`.

Reply:
The comment changes should be in another PR